starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,80 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+.hero {
```

Comment:
it's better, when possible to give a meaningful names to your classes. This is going to be really important for when you have a lot of css

Reply:
hero is the banner image on the webpage! usually at the top of the page/ full width